lib/prometheus/client.rb

```diff
@@ -14,5 +14,13 @@ def self.registry
     def self.config
       @config ||= Config.new
     end
+
+    def self.linear_buckets(start:, width:, count:)
+      count.times.map { |idx| start.to_f + idx * width }
+    end
+
+    def self.exponential_buckets(start:, factor: 2, count:)
+      count.times.map { |idx| start.to_f * factor ** idx }
+    end
```

Comment:
I love this feature, it was actually on our TO-DO list, so glad you tackled it.
I'm not sure about the location of these two methods, though...

Could you move these to be static methods in the `Histogram` class?

Also, should we validate the parameters, like the Go client does?

Reply:
Perhaps validating is an idea? Will have a look at that.

On the class method part, is it worth moving them there? Bearing in mind that would be different from the other clients.

Reply:
I think it's worth moving.
That's where I would look for them as a user of this library. 
And unless I misread the Go code, that's where they are in the Go client too.

Reply:
Ah, I did misread the Go code. I saw it in Histogram.go and assumed it'd be there.
I think the conventions in these two languages are different.
Whereas you do `NewHistogram` in Go, you go `Histogram.new` in Ruby. Everything Histogram related is in the Histogram class itself.
The Client class is only used once for setup on startup, or not at all. It doesn't look like a place where someone would look when creating a histogram.

On that note, though... Could we also add a little section in the `README` documenting these two methods? :D

Reply:
Having them on `Histogram` would definitely match my expectations when writing Ruby. I think going with typical idiom within the language is a very fair shout. In `client_java` they're [chainable methods](https://github.com/prometheus/client_java/blob/ef16c43f63128c14627eeff301a32c12e7751a4e/simpleclient/src/main/java/io/prometheus/client/Histogram.java#L59) on the Histogram builder.